### DISCUSSION TITLE
The Plot's timestamp  (ROS2 Connection) is wrong

### DISCUSSION THREAD

**Discussion by taiyuzhuo** [foxglove/community#71] [Q&A]
It is shown in the below figure that the timestamp is not strictly increase monotonically。
![Screenshot from 2022-02-17 11-21-37](https://user-images.githubusercontent.com/26341668/154398677-cf37e36e-7e43-419f-9431-43e5c89e3fb4.png)

**Comment by bertaveira** [comment]
I am getting the same thing when connecting over ROS2 direct connection. Using rosbridge it does not happen

**Reply by jtbandes** [reply]
Thanks, we are now tracking this issue at https://github.com/foxglove/studio/issues/2862. Please feel free to add any additional info there.
